packages/samples/multiple-types-union/main.cadl

```diff
@@ -0,0 +1,35 @@
+import "@cadl-lang/rest";
```

Comment:
Maybe just name the sample `unions`?

Reply:
I 'll suppress it in cadl-azure, so I think a more specific name is better